src/utils/traffic-mesh.js

```diff
@@ -92,6 +97,67 @@ const startForwardProxy = async ({ port, frameworkPort, functionsPort, publishDi
   }
 }
 
+const forwardMessagesToLog = ({ log, subprocess }) => {
+  let currentId = null
+  let spinner = null
+
+  subprocess.stderr
```

Comment:
Note: we might want to handle `error` events on this stream.

Reply:
Does a pipe error if the child exits unexpectedly? I imagine it'd just return `eof` at some point and that's it.

Reply:
There a few reason why an `error` event would be triggered on a stream. In that case, we might want to do some cleanup of the spinner for example.

Reply:
Sounds fair.